source/features/mark-merge-commits-in-list.tsx

```diff
@@ -36,7 +36,11 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 // eslint-disable-next-line import/prefer-default-export
 export function getCommitHash(commit: HTMLElement): string {
 	return 	commit.dataset.channel!.split(':')[3] ?? // Pre "Repository refresh" layout
-	commit.querySelector<HTMLAnchorElement>('a[href]')!.href.split('/').pop()!;
+	commit.querySelector<HTMLAnchorElement>('a[href]')!
+		.href
+		.split('/')
+		.pop()!
+		.split('#')[0]; // Drop the #diff-;
```

Comment:
Maybe you can use `pathname` so you don't have to split again?

```suggestion
	commit.querySelector<HTMLAnchorElement>('a[href]')!.pathname.split('/').pop()!;
```

Reply:
Yeah that's better